src/app/views/authentication/profile/Profile.tsx

```diff
@@ -106,7 +106,7 @@ export class Profile extends Component<IProfileProps, IProfileState> {
     const persona: IPersonaSharedProps = {
       imageUrl: user.profileImageUrl,
       imageInitials: this.getInitials(user.displayName),
-      text: user.displayName,
+      text: user.displayName + ' ' + (this.props.permissionModeType ? messages['As user'] : messages['As Teams app']),
```

Comment:
I would suggest making `permissionModeType` as an Enum and add maps for conditional display for readability and maintainability. 
```
const enum PermissionModeType {
  User,
  TeamsApp,
}

 const permissionModeTypeDisplayName = {
      [PermissionModeType.User]: 'As user',
      [PermissionModeType.TeamsApp]: 'As Teams app',
    }
```
then you can use
```
text: user.displayName + ' ' + translateMessage(permissionModeTypeDisplayName[this.props.permissionModeType])
```

Reply:
Making a branch to test this out; Thank you for the suggestion!